components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java

```diff
@@ -199,8 +199,18 @@ private Exception createExceptionFromFault(Fault fault) {
                 throw new RuntimeCamelException(e);
             }
         }
+
+	Object detailObj = faultDetail.getAny().get(0);
```

Comment:
There's a tab character here, we have a spaces policy, you can run source checks from Maven:

    $ cd components/camel-soap; mvn -Psourcecheck clean verify

This is what I get now:

    ...
    [INFO] --- maven-checkstyle-plugin:2.17:checkstyle (validate) @ camel-soap ---
    [INFO] Starting audit...
    [ERROR] /home/zregvart/workspaces/camel/camel/components/camel-soap/src/main/java/org/apache/camel/dataformat/soap/Soap11DataFormatAdapter.java:203:1: File contains tab characters (this is the first instance). [FileTabCharacter]
    Audit done.
    ...